src/android/InAppBrowser.java

```diff
@@ -1066,12 +1070,16 @@ public void postMessage(String data) {
                 lp.height = WindowManager.LayoutParams.MATCH_PARENT;
 
                 dialog.setContentView(main);
-                dialog.show();
+                if(dialog != null){
```

Comment:
If `dialog` would be null, it would crash anyway because of the lines above and below. I would put the null check around all of these lines:

```
                if(dialog != null){
                    dialog.setContentView(main);
                    dialog.show();
                   dialog.getWindow().setAttributes(lp);
                }
```

Reply:
And add a space after the `if` and before the `{` for all new guards 🙃

Reply:
Good catch!

Reply:
I just pushed these changes. See my following comment